hackernews/program.py

```diff
@@ -10,18 +10,20 @@
 
 slack = slack_client("slack_connection")
 SLACK_CHANNEL = os.getenv("SLACK_CHANNEL")
+API_URL = "http://hn.algolia.com/api/v1/search_by_date?tags=story&page=0&query="
```

Comment:
you prolly want https here

Reply:
https://hn.algolia.com/api specifies insecure HTTP, but I guess it worth testing. If HTTPS is supported it's better.

Reply:
it worked with https, I changed it